cli/ballerina-cli/src/main/resources/create_cmd_templates/service/tests/service_test.bal

```diff
@@ -0,0 +1,34 @@
+import ballerina/io;
+import ballerina/test;
+
+http:Client testClient = check new ("http://localhost:9090/hello");
+
+# Before Suite Function
+
+@test:BeforeSuite
+function beforeSuiteFunc() {
+    io:println("I'm the before suite function!");
+}
+
+# Test function
+
+@test:Config {}
+function testServiceFunction() returns string {
+    http:Response response = check testClient->get("/sayHello/?name=John");
```

Comment:
The service is written to return a `string`. Therefore the response also should be a `string`